source/enumerate-collections.rst

```diff
@@ -265,7 +266,17 @@ Example return::
         "system.replset"
     ]
 
-Drivers MUST strip the database name from the returned collection names.
+
+
+Drivers MAY implement a MongoClient method that returns an Iterable of strings,
+each of which corresponds to a name in the collections array of the
+``listCollections`` command result. This method SHOULD be named
+``listCollections``.
+
+Older versions of the server that do not support the ``nameOnly`` option for the
+``listCollections`` command will ignore it without raising an error. Therefore,
+drivers SHOULD always specify the ``nameOnly`` option when they only intend to
+access collection names from the ``listCollections`` command result.
```

Comment:
This sentence can remain as-is, since it only applies to `listCollections` command. :+1: